.env.example

```diff
@@ -1,6 +1,13 @@
-REGISTRY="http://<ip>:4000"
+########### server configuration
 EXPRESS_PORT= 4001
 LOCAL_ONLY=1 # Look for the local ip if 1 else, will take the public ip
 DELETE_DB_ON_START=1 # The db should be deleted when start up
 LOCAL_INTERFACES="Ethernet,ens160" # Ethernet card name on your machine
-DATABASE_TYPE="ACEBASE" #IN_MEMORY,ACEBASE 
\ No newline at end of file
+DATABASE_TYPE="ACEBASE" #IN_MEMORY,ACEBASE 
+########### smartcontract definition
+USE_SMART_CONTRACT=1
+REGISTRY="0x3700A8C0447aEE3160F6aF3A34a0C062629335d9" # if USE_SMART_CONTRACT = 0 then it will try a classic http server : "http://<ip>:4000"  
+ABI='[{"inputs":[{"internalType":"contract IERC20","name":"_stakingToken","type":"address"},{"internalType":"uint256","name":"_obligationCost","type":"uint256"}],"stateMutability":"nonpayable","type":"constructor"},{"anonymous":false,"inputs":[{"indexed":true,"internalType":"address","name":"account","type":"address"},{"indexed":false,"internalType":"string","name":"url","type":"string"}],"name":"SetURL","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"internalType":"address","name":"account","type":"address"}],"name":"Stake","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"internalType":"address","name":"account","type":"address"}],"name":"Unstake","type":"event"},{"inputs":[{"internalType":"address","name":"staker","type":"address"}],"name":"balanceOf","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"getURLs","outputs":[{"internalType":"string[]","name":"urls","type":"string[]"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"obligationCost","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"string","name":"_url","type":"string"}],"name":"setURL","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"","type":"address"}],"name":"stakerURLs","outputs":[{"internalType":"string","name":"","type":"string"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"stakingToken","outputs":[{"internalType":"contract IERC20","name":"","type":"address"}],"stateMutability":"view","type":"function"}]'
```

Comment:
current goerli indexer registry is 0xC32a3c867aBAd28d977e1724f92D9684fF3d2976
you can also grab latest indexer registry addresses from @airswap/indexers/deploys.js